added getAllSchema function (#76)
hyperjump-io/json-schema: README.md

* **unregisterSchema**: (uri: string) => void

    Remove a schema from the local schema registry.
* **getAllRegisteredSchemaUris**: () => string[]

>     This function returns all the registered schema URIs present in the repository

Reviewer: ```suggestion
    This function returns the URIs of all registered schemas
```
